src/user/test/unit/user.service.spec.ts

```diff
@@ -93,7 +93,11 @@ describe('UserService', () => {
       .spyOn(prismaService.user, 'create')
       .mockResolvedValue(mockPrismaCreateResult);
 
-    const result = await userService.generateAdminAccount();
+    const result = await userService.generateAccount(
```

Comment:
please add a test case to check if you do not respond with password while creating user